packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -0,0 +1,209 @@
+    if (this.facetInfo || !event.detail) {
+      return;
+    }
+
+    this.facetInfo = event.detail.facetInfo;
+    this.facetInfo.element.classList.add(popoverClass);
+    this.hasValues = event.detail.getHasValues;
+    this.numberOfSelectedValues = event.detail.getNumberOfSelectedValues;
+  }
+
+  private get popoverId() {
+    return `${this.facetInfo?.facetId}-popover`;
+  }
+
+  private togglePopover() {
+    if (!this.isOpen) {
+      this.closePopovers.emit({ownId: this.facetInfo?.facetId});
+    }
+
+    this.isOpen = !this.isOpen;
+  }
+
+  private renderDropdownButton() {
+    const label = this.bindings.i18n.t(this.facetInfo!.label);
+    const hasSelectedValues = !!this.numberOfSelectedValues?.();
+    const count = this.numberOfSelectedValues?.().toLocaleString();
+    const ariaLabel = this.bindings.i18n.t(
+      this.isOpen ? 'close-popover' : 'open-popover',
+      {label}
+    );
+
+    return (
+      <Button
+        style="square-neutral"
+        onClick={() => this.togglePopover()}
+        part="popover-button"
+        ariaExpanded={`${this.isOpen}`}
+        ariaLabel={ariaLabel}
+        ariaControls={this.popoverId}
+        class={`rounded flex box-border h-full items-center min-w-[6rem] max-w-[15rem] p-2.5 group hover:border-primary-light focus-visible:border-primary-light ${
+          this.isOpen
+            ? 'border-primary ring ring-ring-primary text-primary'
+            : ''
+        }`}
+      >
+        <span
+          title={label}
+          part="value-label"
+          class={`truncate mr-0.5 ${
+            this.isOpen
+              ? ''
+              : 'group-hover:text-primary-light group-focus:text-primary'
+          }`}
+        >
+          {label}
+        </span>
+        <span
+          part="value-count"
+          class={`truncate text-sm group-hover:text-primary-light group-focus:text-primary ${
+            hasSelectedValues ? '' : 'hidden'
+          } ${this.isOpen ? 'text-primary' : 'text-neutral-dark'}`}
+        >
+          {this.bindings.i18n.t('between-parentheses', {
+            text: count,
+          })}
+        </span>
+        <atomic-icon
+          part="arrow-icon"
+          aria-hidden="true"
+          class={`w-2 ml-auto group-hover:text-primary-light group-focus:text-primary ${
+            this.isOpen ? 'rotate-180' : ''
+          } `}
+          icon={ArrowBottomIcon}
+        ></atomic-icon>
+      </Button>
+    );
+  }
+
+  public render() {
+    if (this.searchStatus.state.hasError) {
+      return <Hidden></Hidden>;
+    }
+
+    if (!this.searchStatus.state.firstSearchExecuted) {
+      return (
+        <div
+          part="placeholder"
+          aria-hidden
+          class="h-8 w-32 bg-neutral animate-pulse rounded"
+        ></div>
+      );
+    }
+
+    if (!this.searchStatus.state.hasResults || !this.hasValues?.()) {
+      return <Hidden></Hidden>;
+    }
+
+    return (
+      <div id={this.popoverId} class="relative">
```

Comment:
The button has `aria-controls` pointing towards this `id`, however, this element is a parent of the button. I think it would make more sense to move this `id` to the element below which has `part="slot-wrapper"`.